lib/metasploit/framework/login_scanner/ivanti_login.rb

```diff
@@ -0,0 +1,169 @@
+require 'metasploit/framework/login_scanner/http'
+
+module Metasploit
+  module Framework
+    module LoginScanner
+      # Ivanti Login Scanner supporting
+      # - User Login
+      # - Admin Login
+      class Ivanti < HTTP
+
+        def initialize(scanner_config, admin)
+          @admin = admin
+          super(scanner_config)
+        end
+
+        def create_admin_request(username, password, token, protocol, peer)
+          {
+            'method' => 'POST',
+            'uri' => normalize_uri('/dana-na/auth/url_admin/login.cgi'),
+            'ctype' => 'application/x-www-form-urlencoded',
+            'headers' =>
+            {
+              'Origin' => "#{protocol}://#{peer}",
+              'Referer' => "#{protocol}://#{peer}/dana-na/auth/url_admin/welcome.cgi"
+            },
+            'vars_post' => {
+              tz_offset: '60',
+              xsauth_token: token,
+              username: username,
+              password: password,
+              realm: 'Admin+Users',
```

Comment:
Just a question; Was there a reason for preference passing `'encode_params' => false` versus letting the `vars_post` values be encoded by default 👀 

https://github.com/rapid7/metasploit-framework/blob/90ad8b66d8ac470012525d6d25c06e520b434590/lib/rex/proto/http/client_request.rb#L148-L160